core/src/components/input/test/bottom-content/input.e2e.ts

```diff
@@ -105,21 +105,22 @@ configs({ modes: ['md', 'ionic-md'], directions: ['ltr'] }).forEach(({ title, sc
         await expect(errorText).toBeVisible();
         await expect(errorText).toHaveText('my error');
       });
-      test('error text should change when variable is customized', async ({ page }) => {
+      test('error text and highlight should change when variable is customized', async ({ page }) => {
         await page.setContent(
           `
           <style>
             ion-input.custom-input {
               --highlight-color-invalid: purple;
+              --text-color-invalid: purple; // ionic only
```

Comment:
```suggestion
              --text-color-invalid: purple; /* ionic only */
```

It does not appear to have an impact on your test behavior, but the `//` is not the syntax correct comment format for style block in HTML.

![CleanShot 2024-04-11 at 00 13 17@2x](https://github.com/ionic-team/ionic-framework/assets/13732623/36adf886-67bc-4557-b586-7aa9dc359ad6)